src/20240902_AaveV3EthereumEtherFi_EtherFiEthereumActivation/EtherFiEthereumActivation.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "EtherFi Ethereum Activation"
+author: "Catapulta"
+discussions: "https://governance.aave.com/t/arfc-deploy-an-etherfi-stablecoin-aave-v3-instance/18440"
+snapshot: "https://snapshot.org/#/aave.eth/proposal/0x4acd11c6100a6b85a553e21359f3720fa5cd4783a76c77857436ace134f88c05"
+---
+
+## Simple Summary
+
+This AIP activates the EtherFi and Stables Aave V3 instance in Ethereum network, following [ARFQ request](https://governance.aave.com/t/arfc-deploy-an-etherfi-stablecoin-aave-v3-instance/18440).
+
+## Motivation
+
+The demand for borrowing wETH on Aave using weETH as collateral is extremely high, with caps being filled within minutes each week.
+
+While this $2B growth is welcomed and responsible for ~$12.5M yearly revenue (at current ETH prices) for the DAO, it has created some frustration for significant users looking to borrow stablecoins using their weETH as collateral.
+
+With this asset being constantly at cap, users are wary of leveraging Aave for this use case as they have legitimate concerns about their ability to add more collateral/debt to (de)leverage following market conditions.
+
+To provide weETH holders with more suitable options, we propose creating a dedicated weETH/stablecoin Aave v3 Instance.
+
+This market also lays the groundwork for future integrations & synergies with EtherFi’s Cash product.
+
+## Specification
+
+The Etherfi/Stables Aave v3 instance activation payload implements the following:
+
+- Pre-execution:
+  - Register Ethereum EtherFi instance in Ethereum V3 PoolAddressesProviderRegistry with ID 45.
+- Execution:
+  - Listing of weETH, USDC, PYUSD, and FRAX tokens.
+- Post execution:
+  - Add PROTOCOL_GUARDIAN as POOL_ADMIN role
+  - Add CAPS_PLUS_RISK_STEWARD as RISK_ADMIN role
+  - Seed amounts
+  - Pay Catapulta service fee
+
+The table below illustrates the configured risk parameters for **weETH**
+
+| Parameter                 |                                      Value |
```

Comment:
@kartojal i would add all the assets in a single table as its a bit easier to read

Reply:
done!